documents/postman/staging-simplified.postman_environment.json

```diff
@@ -0,0 +1,71 @@
+{
+	"id": "1acffa98-51c5-4ba7-87fc-9f3d460c38ba",
+	"name": "development Copy",
+	"values": [
+		{
+			"key": "email-from",
+			"value": "solutions",
+			"enabled": true
+		},
+		{
+			"key": "email-template-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "notification-api-url",
+			"value": "https://staging-api.va.gov/vanotify",
+			"enabled": true
+		},
+		{
+			"key": "notification-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "service-api-key",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "service-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "service-name",
+			"value": "VA Notify",
+			"enabled": true
+		},
+		{
+			"key": "sms-sender-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "sms-template-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "template-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "api-key-id",
```

Comment:
Is this variable needed/used here? If so - for which endpoint in this simplified collection?

Reply:
It's used by Postman to populate the POST body.

Reply:
`api-key-id` not used and should be removed.

`service-api-key` is the one used for the POST request

Reply:
@edDocMe360 see Cris's comment above - I just sent an email to myself without it api key id populated and it worked/is still blank in the env variables. Can you re-check and remove if not needed?